Library/Formula/blackbox.rb

```diff
@@ -0,0 +1,36 @@
+class Blackbox < Formula
+  homepage "http://the-cloud-book.com/"
+  url "https://github.com/StackExchange/blackbox/archive/v1.20150730.tar.gz"
+  desc "Safely store secrets in Git/Mercurial/Subversion"
+  sha256 "a28237e8839d000273d0fe0ba8f9701cab66a3b942787c5c8e8d329912564ae4"
+
+  depends_on "gpg"
```

Comment:
This should be `:gpg` unless it can't use `gnupg2`, in which case it should be `gnupg`.